src/components/TracePage/TraceTimelineViewer/Ticks.js

```diff
@@ -18,42 +20,52 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import PropTypes from 'prop-types';
-import React from 'react';
+import * as React from 'react';
+
+import { formatDuration } from './utils';
 
 import './Ticks.css';
 
-export default function Ticks(props) {
-  const { labels, ticks } = props;
+type TicksProps = {
```

Comment:
Seeing a mixture of flowTypes and propTypes in this review. Let's stick with one

Reply:
The `PropTypes` are holdovers from before. I've moved all of `TraceTimelineViewer/*` to use flow.

Reply:
Ok gotcha